max_x: 30 # [m]
max_y: 50 # [m]
max_z: 10 # [m]
